src/GroupRow.tsx

```diff
@@ -83,8 +74,7 @@ function GroupedRow<R, SR>({
           groupRowClassname,
           `rdg-row-${rowIdx % 2 === 0 ? 'even' : 'odd'}`,
           {
-            [rowSelectedClassname]: isRowFocused,
-            [rowSelectedWithFrozenCell]: isRowFocused && lastFrozenColumnIndex !== -1
+            [rowSelectedClassname]: selectedCellIdx === -1
```

Comment:
This does not add any styles but without this class, there is no indication of the row selected. I wonder if there is any aria attribute